test/trusted_resources_test.go

```diff
@@ -223,13 +223,220 @@ spec:
 
 }
 
-func setupResourceVerificationConfig(ctx context.Context, t *testing.T, fn ...func(context.Context, *testing.T, *clients, string)) (*clients, string, string, signature.Signer) {
+func TestTrustedResourcesVerify_VerificationPolicy_Success(t *testing.T) {
```

Comment:
Yes this is mainly to make sure we could fetch public keys from `VerificationPolicy`.  It may be possible to update the existing tests. I will try and see if we can merge them. Thanks for the suggestion!